Sources/Controllers/2.0/BottomSheet/OABaseBottomSheetViewController.m

```diff
@@ -117,7 +119,17 @@ - (CGFloat)initialHeight
 
 - (CGFloat) buttonsViewHeight
 {
-    return 60.;
+    if ([OAUtilities isLandscape])
+    {
+        CGFloat bottomPadding = [OAUtilities getBottomMargin];
```

Comment:
[OAUtilities getBottomMargin] takes orientation into account, so no additional check is needed